### PR TITLE
Feature/common sessions fragment

### PR DESCRIPTION
**Pull request by pps5**
## Overview (Required)
- Use BottomSheetSessionsFragment for all session tabs
  - Different fragments between for days and for favorite lead to omit changing
  - Fragments have almost same functionality
- Small fix to make empty sessions layout responsive

## Links
- https://github.com/DroidKaigi/conference-app-2020/blob/75b7b342ac0327ad67763bf426acf5cde70652af/feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetDaySessionsFragment.kt#L121

### CODE REVIEW

**Comment by takahirom** [comment]
👀

**Comment by takahirom** [comment]
I'm sorry to be late though conflict is easy to occur. 🙏

**Comment by takahirom** [comment]
I agree with the policy because there are many bugs where the display is not the same.

**Comment by takahirom** [comment]
Can you fix conflict ?🙇

**Comment by pps5** [comment]
@takahirom 
Sorry for late.
I fixed conflicts, but GitHub actions failed by timeout. What can I do about it?

**Comment by takahirom** [comment]
now rerun 👍

File: feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml [hunk 1]
```diff
@@ -91,5 +112,46 @@
...
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:isInvisible="@{isCollapsed || !isEmptyFavoritePage || isFiltered}"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            >
+
+            <ImageView
```
[the remark this answers is not in the document]
Comment (pps5) [reply feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml]: My bad, I added this file when fixing conflicts...
I'll remove this file again later.

[the remark this answers is not in the document]
Comment (takahirom) [reply feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml]: 🙏

[the remark this answers is not in the document]
Comment (pps5) [reply feature/session/src/main/res/layout/fragment_bottom_sheet_sessions.xml]: completed

**Comment by takahirom** [comment]
LGTM!!Thanks for code that can reduce bugs!

**Pull request merged**